src/instantsend.cpp

```diff
@@ -529,13 +529,11 @@ void CInstantSend::UpdateLockedTransaction(const CTxLockCandidate& txLockCandida
     if (!IsLockedInstantSendTransaction(txHash)) return; // not a locked tx, do not update/notify
 
 #ifdef ENABLE_WALLET
-    if (pwalletMain && pwalletMain->UpdatedTransaction(txHash)) {
```

Comment:
Removing this call will break the UI for InstantSend...we need to find an alternative way to make it call `NotifyTransactionChanged`. One option would be to reintroduce `CWallet::UpdatedTransaction` but name it differently, e.g. `NotifyTransactionChangedIfWalletTx`.

Reply:
Or we can do smth like 63310712e.

Reply:
cherry-picked

Reply:
:+1: on https://github.com/dashpay/dash/commit/63310712e5aed5f67a38d176068a46746fcf7b27